packages/cli/internal/pkg/cli/configure_format.go

```diff
@@ -30,8 +30,8 @@ func newFormatContextOpts(vars formatContextVars) (*formatContextOpts, error) {
 	}, nil
 }
 
-func (o *formatContextOpts) Validate() error {
-	if o.formatContextVars.format == "" {
+func (o *formatContextOpts) Validate(args []string) error {
```

Comment:
Should we be passing an array in here if we're only reading `args[0]` out of it later on?

Reply:
That function is handling validation which also checks if the array is empty to provide users with a readable error message